src/Components/CroppedThumbnail/CroppedThumbnail.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import { Image, ImageProps, ImageURISource } from "react-native";
+
+type CropMode = "c_scale" | "c_fit" | "c_crop";
+
+type FetchFormat = "f_auto" | "f_png";
+
+interface IOptions {
+  fetch_format?: FetchFormat;
+  crop?: CropMode;
+  width?: number;
+  height?: number;
+  rWidth?: number;
+  rHeight?: number;
+  face?: "g_face";
+}
+
+interface IImageTransformationOptions {
+  fetch_format?: IOptions["fetch_format"];
+  crop?: IOptions["crop"];
+  width?: string;
+  height?: string;
+}
+
+export const croppedThumbnailDefaults: IOptions = {
+  fetch_format: "f_auto",
+  crop: "c_fit",
+};
+
+type KeyofOptions = keyof IOptions;
+type CroppedThumbnailProps = Omit<ImageProps, "source">;
+
+interface IProps extends CroppedThumbnailProps {
+  accessibilityLabel: string;
+  imageId: string;
+  options: {
+    fetch_format?: "f_auto" | "f_png";
```

Comment:
It is better to move those objects to the upper level to make component usage more fluet

Reply:
It will make a code a little bit more difficult inside the module, but you should provide easier usage of your component its' consumers